ui/src/app/utils/cdr-versions.spec.tsx

```diff
@@ -1,47 +1,90 @@
 import * as React from "react";
 
-import {Workspace, CdrVersion, CdrVersionListResponse} from 'generated/fetch';
+import {Workspace} from 'generated/fetch';
 import {WorkspaceStubVariables} from 'testing/stubs/workspaces';
-import {cdrVersionListResponse} from 'testing/stubs/cdr-versions-api-stub';
-import {getCdrVersion, getDefaultCdrVersion, hasDefaultCdrVersion} from "./cdr-versions";
+import {cdrVersionTiersResponse} from 'testing/stubs/cdr-versions-api-stub';
+import {getCdrVersion, hasDefaultCdrVersion, getDefaultCdrVersionForTier} from "./cdr-versions";
```

Comment:
nit: partially an existing issue, but please qualify these from the project root